de/faq/guide_lecturer.html

```diff
@@ -0,0 +1,436 @@
+---
+title: User Guide - Dozent
+layout: tour.de
+---
+
+<!DOCTYPE html>
+<html lang="de">
+	<head>
+	
+	</head>
+	<body>
+ 			<section class="tour-section">
+ 				<div class="centered">
+ 					<h1 class="hN">1. Nutzung von PINGO als Dozent</h1>
+ 					<h2>Bitte beachten Sie, dass die Dozentensicht für aktuelle Versionen von 		Mozilla Firefox, Google Chrome und Apple Safari optimiert wurde.</h2>
+ 					<div class="grid grid--rev vertical-center">
+ 						<div class="grid__item one-half portable--one-whole cf fullsize">
+ 						
+ 						</div>
+ 						<div class="grid__item one-half portable--one-whole">
+ 							<p></p>
+ 						</div>
+ 					</div>
+ 				</div>
+ 			</section>
+			<section class="tour-section tour-section--odd">
+				<div class="centered">
+					<h1 class="hN">1.1 Ein Konto anlegen</h1>
+					<h2>Hinweis: Dieser Schritt ist nur erforderlich, wenn Sie noch keine Zugangsdaten von uns erhalten haben.</h2>
+					<div class="grid grid--rev vertical-center">
+						<div class="grid__item one-half portable--one-whole cf fullsize">
+							<figure><img class="guideimg" src="/assets/img/guide/registrierung.PNG" alt="Devices" class="float--right portable--float-none"></figure>
+						</div>
+						<div class="grid__item one-half portable--one-whole">
+							<p>
+	   Zur Nutzung der Applikation ist ein Benutzerkonto erforderlich, welches unter pingo.upb.de/users/sign_up erstellt werden kann. Hier müssen nun die für die Anmeldung erforderlichen Daten angegeben werden. Nachfolgend wird die Registration mit einem Klick auf "Registrieren" abgeschlossen.</p>
+						</div>
+					</div>
+				</div>
+			</section>
+			
+ 			<section class="tour-section">
+ 				<div class="centered">
+ 					<h1 class="hN">1.2 Als Dozent anmelden</h1>
+ 					<h2>Um zur Dozentensicht zu gelangen, ist ein Benutzerkonto notwendig (für die Erstellung eines Benutzerkontos siehe Abschnitt 1.1). Öffnen Sie pingo.upb.de und klicken Sie in der Navigationsleiste auf "Einloggen". Geben Sie anschließend Ihre Nutzerdaten ein.</h2>
```

Comment:
So wie hier ist an ein zwei Stellen noch Fließtext in nem `h2` Tag, das liest sich nicht so angenehmen:
![bildschirmfoto 2017-01-24 um 13 48 59](https://cloud.githubusercontent.com/assets/51182/22247846/28305a8c-e23c-11e6-8174-16eedc64f7f8.png)

Evtl. geht man das aber heute Abend oder zu zweit mal durch und behebt sowas... Ebenso könnte man `pingo.upb.de` passend mit HTTPS verlinken, etc.